components/arm/arm.go

```diff
@@ -178,10 +180,10 @@ func CheckDesiredJointPositions(ctx context.Context, a Arm, desiredJoints []floa
 	}
 	checkPositions := currentJointPos.Values
 	model := a.ModelFrame()
-	joints := model.ModelConfig().Joints
+	limits := model.DoF()
```

Comment:
[q] is `limits[i]` guaranteed to be in rad?

Reply:
This is fixed.